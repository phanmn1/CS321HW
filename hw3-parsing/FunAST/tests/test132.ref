Function
  int
  "main"
  Block
    ExprStmt
      Call
        "hello"
        Add
          IntLit(1)
          IntLit(2)
        LAnd
          IntLit(3)
          Id("b")
    ExprStmt
      Call
        "f"
        Call
          "g"
          Id("x")
          Id("y")
