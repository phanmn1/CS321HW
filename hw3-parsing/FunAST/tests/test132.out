Invalid syntax at (1,4), hello
